frontend/src/app/testResults/EmailTestResultModal.tsx

```diff
@@ -0,0 +1,81 @@
+import Modal from "react-modal";
+import moment from "moment";
+
+import Button from "../commonComponents/Button/Button";
+import { formatFullName } from "../utils/user";
+import "./TestResultCorrectionModal.scss";
+import {
+  useGetTestResultForResendingEmailsQuery,
+  useResendTestResultsEmailMutation,
+} from "../../generated/graphql";
+import { showAlertNotification } from "../utils";
+import "./EmailTestResultModal.scss";
+
+const formatDate = (date: string | undefined) => {
+  return moment(date)?.format("MMMM Do, YYYY");
+};
+
+interface Props {
+  testResultId: string;
+  closeModal: () => void;
+}
+
+export const EmailTestResultModal = ({ closeModal, testResultId }: Props) => {
+  const [emailTestResult] = useResendTestResultsEmailMutation();
+
+  const { data, loading } = useGetTestResultForResendingEmailsQuery({
+    variables: { id: testResultId },
+    fetchPolicy: "no-cache",
+  });
+
+  const patient = data?.testResult?.patient as User;
+  const dateTested = data?.testResult?.dateTested;
+  const patientLinkId = data?.testResult?.patientLink?.internalId || "";
+
+  return (
+    <Modal
+      isOpen={true}
+      className="email-test-result-modal-content"
+      overlayClassName="sr-test-correction-modal-overlay"
+      contentLabel="Printable test result"
+    >
+      <div className="header">Email result?</div>
+      {loading ? (
+        <p>Loading</p>
+      ) : (
+        <>
+          <div className="body">
+            <div className="text">
+              {formatFullName(patient)}'s test result from{" "}
+              {formatDate(dateTested)} will be sent to the following emails:
+            </div>
+            <div>{patient?.email}</div>
```

Comment:
What would you think about being a little more defensive against patients with no email addresses? The modal such users will get doesn't provide much value:

<img width="395" alt="Screen Shot 2021-11-05 at 4 09 01 PM" src="https://user-images.githubusercontent.com/27730981/140572707-20874897-58f8-4b28-8f36-6af59441e021.png">

At the very least I'd advocate for updating the modal to be explicit that the selected patient has no emails, but it may also make sense to gray out/remove the `Email results` option entirely for those users. This isn't specified at all in the ticket, so I'd say it's dealer's choice unless @kenieh has an opinion.

Reply:
Great find!

Reply:
It will not show the `Email result` action, when a patient doesnt have an email. we can alter this behavior per @kenieh suggestion when they come back.